metadata-jobs/mae-consumer/src/test/java/com/linkedin/metadata/kafka/hook/spring/MCLSpringTest.java

```diff
@@ -16,16 +17,17 @@
 
 
 @SpringBootTest(classes = {
-    MCLSpringTestConfiguration.class
+    MCLSpringTestConfiguration.class, ConfigurationProvider.class
```

Comment:
Guessing this was the main change from the other PR?

Reply:
Some additional test beans, and the selection of the schema registry implementation in the test.